.gitignore

```diff
@@ -8,3 +8,6 @@ screenshots
 packages/gestalt/dist/*
 # the flow export file as it's maintained by hand
 !packages/gestalt/dist/gestalt.js.flow
+
+# exclude vim files
+*.sw*
```

Comment:
This is local config that you should probably keep in your machine's global gitignore. Gestalt doesn't support local editor setups.

Reply:
I'll remove this commit 👍